.github/pull_request_template.md

```diff
@@ -19,6 +19,12 @@ Tips
 ----
 
 - If the validation check fails:
-  1. Run `make check-style` (from the root directory of the repository) and fix any [flake8](http://flake8.pycqa.org) errors.
-  2. Run `make format` to format your code with the [black](https://black.readthedocs.io/en/stable/index.html) autoformatter.
+
+  Check [style guidelines](https://mitiq.readthedocs.io/en/stable/contributing.html#style-guidelines) for Mitiq.
```

Comment:
Put this after steps 1 and 2. If they need more background they can check out this page, but it shouldn't be listed as a step.